tests/ui/check-cfg/raw-keywords.edition2021.stderr

```diff
@@ -14,7 +14,7 @@ warning: unexpected `cfg` condition name: `r#false`
 LL | #[cfg(r#false)]
    |       ^^^^^^^
    |
-   = help: expected names are: `r#async`, `clippy`, `debug_assertions`, `doc`, `doctest`, `edition2015`, `edition2021`, `fmt_debug`, `miri`, `overflow_checks`, `panic`, `proc_macro`, `relocation_model`, `rustfmt`, `sanitize`, `sanitizer_cfi_generalize_pointers`, `sanitizer_cfi_normalize_integers`, `target_abi`, `target_arch`, `target_endian`, `target_env`, `target_family`, `target_feature`, `target_has_atomic`, `target_has_atomic_equal_alignment`, `target_has_atomic_load_store`, `target_os`, `target_pointer_width`, `target_thread_local`, `target_vendor`, `r#true`, `ub_checks`, `unix`, and `windows`
+   = help: expected names are: `r#async`, `edition2015`, `edition2021`, and `r#true` and 30 more
```

Comment:
This double `and ... and 30 more` is unfortunate *(and pre-existing)*. [`DiagSymbolList`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_errors/struct.DiagSymbolList.html) doesn't gives us enough control to says that the list has been truncated by X amount, and even if it did, [`icu_list`](https://docs.rs/icu_list/latest/icu_list/index.html) doesn't seems to have a way to say that the list has been truncated either.

We could hack it around in check-cfg by adding a fake "X more" in the list. Idk if it's worth it.

Reply:
I think this is totally fine